src/anomalib/data/kolektor.py

```diff
@@ -0,0 +1,179 @@
+from __future__ import annotations
```

Comment:
The file should include the source to the original dataset and a license header. You can have a look here https://github.com/openvinotoolkit/anomalib/blob/main/src/anomalib/data/mvtec.py